webrtc.html

```diff
@@ -153,8 +153,8 @@ <h3>Introduction</h3>
       exchange of control messages (called a signaling protocol) over a
       signaling channel which is provided by unspecified means, but generally
       by a script in the page via the server, e.g. using
-      <code>XMLHttpRequest</code> [[XMLHttpRequest]] or Web Sockets
-      [[WEBSOCKETS-API]].</p>
+      <code>XMLHttpRequest</code> [[?XMLHttpRequest]] or Web Sockets
```

Comment:
What's the difference between [[Foo]], [[\Foo]], [[!Foo]] and [[?Foo]]?

Reply:
` [[Foo]]` is a reference whose status (normative vs informative) inherits from the status of its context (normative in a normative section, informative in a non-normative one).
`[[!Foo]]` used to allow to distinguish normative from informative, but I think it no longer is needed (and would create a warning in an informative section).
`[[?Foo]]` allows to force informative references in a normative section.
`[[\Foo]]` is an internal slot named  `Foo`.